packages/i/input-remapper/pspec_x86_64.xml

```diff
@@ -0,0 +1,209 @@
+<PISI>
+    <Source>
+        <Name>input-remapper</Name>
+        <Homepage>https://github.com/sezanzeb/input-remapper</Homepage>
+        <Packager>
+            <Name>Joey Riches</Name>
+            <Email>josephriches@gmail.com</Email>
+        </Packager>
+        <License>GPL-3.0-or-later</License>
+        <PartOf>system.utils</PartOf>
+        <Summary xml:lang="en">An easy to use tool to change the behaviour of your input devices.</Summary>
+        <Description xml:lang="en">An easy to use tool to change the behaviour of your input devices. Supports X11, Wayland, combinations, programmable macros, joysticks, wheels, triggers, keys, mouse-movements and more. Maps any input to any other input.
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>input-remapper</Name>
+        <Summary xml:lang="en">An easy to use tool to change the behaviour of your input devices.</Summary>
+        <Description xml:lang="en">An easy to use tool to change the behaviour of your input devices. Supports X11, Wayland, combinations, programmable macros, joysticks, wheels, triggers, keys, mouse-movements and more. Maps any input to any other input.
+</Description>
+        <PartOf>system.utils</PartOf>
+        <Files>
+            <Path fileType="config">/etc/dbus-1/system.d/inputremapper.Control.conf</Path>
+            <Path fileType="config">/etc/xdg/autostart/input-remapper-autoload.desktop</Path>
```

Comment:
Are these the correct directories? XDG autostart at least I'm sure has a stateless directory.

Reply:
oh you're right, thanks